src/shared/auth-context.ts

```diff
@@ -5,11 +5,15 @@ export interface User {
     id: string | undefined;
     displayName: string | null;
     photoURL: string | null;
+    expectedDueDate: string | null;
+    setExpectedDueDate: any;
```

Comment:
Need to look into the type definition here so can better understand the structure of it and not use any